printer_zpl2/models/printing_label_zpl2.py

```diff
@@ -64,7 +64,6 @@ class PrintingLabelZpl2(models.Model):
     action_window_id = fields.Many2one(
         comodel_name="ir.actions.act_window",
         string="Action",
-        readonly=True,
```

Comment:
Why removing this? 🤔

Reply:
I've moved that attribute in the form view

Reply:
> If a field is marked as readonly=True on the model, it won't be possible to import it through the Odoo import tool, so avoid it as possible and define the readonly attribute on the views instead.

https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0

Reply:
I wrote it, because it was arisen in several PRs, and it's OK to remove in general `readonly=True` attributes from fields and move them to views. The text is already saying about that: if you need to import by whatever reason records from that model, you won't be able to import the fields marked as `readonly=True` in the fields, reducing the possibilities of the import because they can't be imported and leading to imported records with half of the data. You can't assume that they are going to be computed/filled somehow on create/write, because that's not the usual thing. That said, the sentence is correct because it's not forbidding it, so you judge if it makes sense to keep the `readonly=True` or not.